packages/server/modules/workspaces/services/invites.ts

```diff
@@ -223,7 +228,16 @@ export const collectAndValidateWorkspaceTargetsFactory =
       await deps.validateWorkspaceMemberProjectRoleFactory({
         workspaceId,
         userId: targetUser.id,
-        projectRole
+        projectRole,
+        workspaceAccess: workspaceRoleAndSeat
```

Comment:
fixed issue where invites were not validating against the planned role & seat (we were only checking against the real/existing one)